packages/core/src/Properties.ts

```diff
@@ -67,6 +60,26 @@ export class Properties {
     );
   }
 
+  private getStoredLanguage() {
+    if (typeof localStorage !== 'undefined') {
+      const storedLanguage = localStorage.getItem(
+        CURRENT_LANGUAGE_LOCAL_STORAGE_KEY
+      );
+
+      if (this.config.availableLanguages) {
+        const isSavedLanguageAvailable =
+          this.config.availableLanguages.indexOf(storedLanguage) > -1;
+        // use stored language only if its in available languages
+        if (isSavedLanguageAvailable) {
+          return storedLanguage;
+        }
+      } else {
+        // no available languages specified
```

Comment:
This is also obvious.

Reply:
It would be more readable if you returned it if(!this.config.availableLanguages) and omitted the else.

Reply:
Hmm, I didn't find this whole process of selecting language that obvious as there is a case when there is a stored language but doesn't fit available languages and therefore is not used - because of that I rather commented what exactly these branches mean

Reply:
But yes the negative branch is more readable